.ci/update_translations_template.md

```diff
@@ -1,13 +1,11 @@
-After merging, all changes to the source language are available for translation at [Transifex](https://www.transifex.com/projects/p/cockatrice/) shortly.
-
-<br>
-
 Updated language source strings:
-- {{ .cockatrice_output }} in cockatrice
-- {{ .oracle_output }} in oracle
+- {{ .cockatrice_output }} (Cockatrice)
+- {{ .oracle_output }} (Oracle)
 
 <br>
 
-The most recent update is based on commit: {{ .commit }}
+Last changes are based on commit {{ .commit }}.
 
-*This PR is automatically generated and updated by the workflow at `.github/workflows/translations.yml`. This template is located at `.ci/update_translations_template.md`.*
+---
+*After merging, all changes to the source language are available for translation at [Transifex](https://www.transifex.com/projects/p/cockatrice/) shortly.*<br>
```

Comment:
I say the text this pr is automatically generated should be the first part of this section